File: 21-01-20-pclassic/p1/Perfect.py
import math


def is_perfect(n) -> bool:
    # Your implementation here!
    return False


if __name__ == '__main__':
    with open('PerfectIN.txt', 'r') as f:
        while True:
            n = f.readline()
            if n == '':
                break
            n = int(n)
            out = is_perfect(n)
            if out:
                print('true')
            else:
                print('false')

File: 21-01-20-pclassic/p1/Perfect.java
import java.io.*;
import java.util.*;

public class Perfect {
  public static void main(String[] args) throws IOException {

    Scanner sc = new Scanner(new FileReader("PerfectIN.txt"));
    while (sc.hasNext()) {
      System.out.println(isPerfect(sc.nextInt()));
    }

  }

  public static boolean isPerfect(int n) {
    // Your implementation here!
  }
}
